docs/getting-started/tutorial.md

```diff
@@ -0,0 +1,382 @@
+
+There's nothing else on this page because we haven't added any questions yet! Let's do that next.
+
+### Step 6: Adding a question
+
+For the purposes of this tutorial, we'll be building a simple address form. The first question when asking for an address is typically the street, so let's start by adding a field for street within the `schema: { properties: {}}` object we just created.
+
+```js
+...
+  schema: {
+    type: 'object',
+    properties: {
+      street: {
+        type: 'string'
+      }
+    }
+  }
+...
+```
+
+Let's see what the form looks like now that we've added this question. The webpack server should have automatically refreshed the page with your form, so go back to your browser to take a look at what's there.
+
+This is what you should see:
+![Street field](../images/street-field.png)
+
+Hooray, a form field has appeared! But how did the library know to render a text input with a label of `street` based on what we added to the `schema`?
+
+The data that the `street` field accepts is `string` data. For the basic data types, an automatic determination is made by the library code on what type of HTML form element to render. `string` data renders a text input, a field with an `enum` property renders a select, and `boolean` data renders checkbox.
+
+The label is taken from the name of the field in the `formConfig`. `street` becomes "street" in our form. But what if we want to change the name that appears to the user? Let's do that next.
+
+### Step 7: Adding the `uiSchema` object.
+
+While the `schema` is the main description of the fields of our form, there are UI-specific things we may want to change about how those fields appear. Remember, the `schema` is mainly just a description of the type of `data` of that field, and not as concerned with how that field is displayed to the user.
+
+That's where `uiSchema` comes in. You can think of the `uiSchema` object as a mirror to the `schema` one. It usually has the same fields as the `schema` object, with different properties within each field to specify UI-specific characteristics of that field.
+
+The `uiSchema` object is added to our `page` at the same level as the `schema` object:
+
+```js
+firstPage: {
+  path: 'first-chapter/first-page',
+  title: 'First Page',
+  schema: {
+    type: 'object',
+    properties: {
+      ...
+    }
+  },
+  uiSchmea: {
+
+  }
+}
+```
+
+Let's return to the example of changing the label text of the `street` field. We can pass a property to the `uiSchema` under `street` called `ui:title` that lets us specify the exact label text we want displayed. This is how your `formConfig` would look:
+
+```js
+...
+  schema: {
+    type: 'object',
+    properties: {
+      street: {
+        type: 'string'
+      }
+    }
+  },
+  uiSchema: {
+    street: {
+      'ui:title': 'Street'
+    }
+  }
+...
+```
+(Note: most properties that are passed to `uiSchema` are prefixed with `ui:`, and therefore need to be passed as a string to `formConfig`.)
```

Comment:
I think this edit would be leaving out helpful context on why those properties are prefixed, and how that's a common pattern.

Reply:
Then what about:

"Some properties that are passed to `uiSchema` are prefixed with `ui:`. Because of this, they must be passed as a string to `formConfig`."

The thing I'm trying to avoid is the leap to "therefore". It's not necessarily obvious that the latter is because of the former, and "therefore" implies it is.